src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/FixedDocument.cs

```diff
@@ -836,7 +829,7 @@ private void _Init()
         {
             _paginator = new FixedDocumentPaginator(this);
             _pages = new PageContentCollection(this);
-            _highlights = new Dictionary<FixedPage, ArrayList>();
+            _highlights = new Dictionary<FixedPage, ArrayList>(); // todo: get rid of ArrayLists
```

Comment:
What's the point of adding this todo? Isn't it more useful to open an issue?

Reply:
Docs say that [ArrayList aren't recommended and shouldn't be used anymore](https://learn.microsoft.com/en-us/dotnet/api/system.collections.arraylist?view=net-7.0#remarks). But I agree that it makes more sense to track this in a separate issue and leave comments like this out of code as they only confuses people.